src/components/Input/index.tsx

```diff
@@ -15,13 +14,23 @@ interface Props {
   onChange: () => void;
   onFocus: () => void;
   onKeyDown: () => void;
+  isValid: (value) => boolean;
   placeholder?: string;
   size: Size;
-  state?: State;
   value?: string;
   variablesClassName?: string;
+  validationRegex?: string;
```

Comment:
please add a default for this prop. Something like `.*` should be enough

Reply:
@kaiomagalhaes, I added the default regex here https://github.com/codelittinc/agnostic-design-system/pull/65/files#diff-fcd026843206e6f1e311fdf284f9cace927b8cf80dd11a72849d331b32aa5e16R32